src/App.tsx

```diff
@@ -52,15 +53,45 @@ function App() {
     setBoardValues(["-", "-", "-", "-", "-", "-", "-", "-", "-"]);
   }
 
+  const computerTurn = () => {
+    // Creating copy of the board
+    let cloneBoardValues = [...boardValues];    
```

Comment:
Do we need to clone here? Since we're not modifying `cloneBoardValues`, but just reading the values, it seems okay to just use `boardValues` below